astropy/io/fits/hdu/hdulist.py

```diff
@@ -1057,7 +1065,8 @@ def _readfrom(cls, fileobj=None, data=None, mode=None, memmap=None,
         if fileobj is not None:
             if not isinstance(fileobj, _File):
                 # instantiate a FITS file object (ffo)
-                fileobj = _File(fileobj, mode=mode, memmap=memmap, cache=cache)
+                fileobj = _File(fileobj, mode=mode, memmap=memmap,
+                                cache=cache, **kwargs)
```

Comment:
Adding `**kwargs` to `_File` enables the `use_fsspec` and `fsspec_kwargs` parameters to reach `utils.get_readable_fileobj()`.